edge/pkg/metamanager/msg_processor.go

```diff
@@ -105,7 +115,7 @@ func parseResource(resource string) (string, string, string) {
 
 // is resource type require remote query
 func requireRemoteQuery(resType string) bool {
-	return resType == model.ResourceTypeConfigmap || resType == model.ResourceTypeSecret
+	return resType == model.ResourceTypeConfigmap || resType == model.ResourceTypeSecret || resType == constants.ResourceTypeService || resType == constants.ResourceTypeEndpoints
```

Comment:
why we need edgemesh answer the remote query for those two resources?

Reply:
Suppose an endpoint or service is not found in edge, they can be queried from cloud